test/fixtures/async-app/app.js

```diff
@@ -0,0 +1,21 @@
+var loopback = require('loopback');
+var boot = require('loopback-boot');
+
+var app = loopback();
+
+// Listen on an ephemeral port
+app.set('port', 0);
+
+// Setup default datasources for autoAttach()
+app.dataSource('db', { connector: 'memory', defaultForType: 'db' });
+app.dataSource('mail', { connector: 'mail', defaultForType: 'mail' });
+
+// Attach all built-in models
+loopback.autoAttach();
```

Comment:
`autoAttach` is deprecated and will be removed in LB 3.0. Please remove L9-L14 and configure built-in models via `test/fixtures/async-app/model-config.json` instead. Since your test does not rely on any built-in models, I think you can simply remove this few lines and do not add model-config.json at all.